openghg/util/_data_level.py

```diff
@@ -0,0 +1,58 @@
+from typing import Optional, overload
+import logging
+from openghg.types import MetadataFormatError
+
+__all__ = ["format_data_level"]
+
+logger = logging.getLogger("openghg.util")
+logger.setLevel(logging.INFO)  # Have to set level for logger as well as handler
+
+
+@overload
+def format_data_level(
```

Comment:
You could add int and float to the accepted inputs, the output will be str in any case. (With corresponding change in the actual implementation.) Not a big deal, I will probably look for things like that in the metadata formatting updates.